nncf/openvino/quantization/helpers.py

```diff
@@ -0,0 +1,106 @@
+"""
```

Comment:
Move functions from here to specialized files even if it means having one function to a file for now. The `_convert` functions may go to the `nncf/openvino/converters.py`, either as-is or consolidated into a `Converter` object or something like that, `quantize_impl` may go to `nncf/openvino/quantization/impl.py`

Reply:
The `_convert_openvino_model_to_compressed_model()` and `_convert_compressed_model_to_openvino_model` methods are used only inside the `nncf.openvino.quantization.helpers` module. Of course, it is better to create a separate module if these methods are needed in other places. But at the moment it's not like that. Probably these methods will be removed in the future.